### ISSUE TITLE
Make a release

### ISSUE THREAD

**Issue by schmod** [yeoman/grunt-usemin#561]
The dev branch contains several new and notable features.  

Specifically, the completion of #316 and #518 (throwing errors when resources can't be resolved) is a really big deal.

What needs to be done before we can cut a release?

**Comment by sindresorhus** [comment]
It needs to be completed and merged into master.

//@stephanebachelier

**Comment by schmod** [comment]
What remains to be done?  I'd be happy to lend a hand if there's a TODO list.

**Comment by alippai** [comment]
+1

**Comment by arthurvr** [comment]
> What remains to be done? I'd be happy to lend a hand if there's a TODO list.

We welcome any contributions! The more the merrier ;) I guess the todo list at the moment is basically the ever growing issue list. Triaging issues, reviewing PRs or simply testing the `dev` branch is helpful.

**Comment by stephanebachelier** [comment]
I'll have some time this week to work on dev branch. 
It would be helpful to test the dev branch in your workflow and in majors yeoman generators.

**Comment by marcelaraujo** [comment]
@arthurvr  @stephanebachelier If you need some help to complete this task like testing new bugs opened after my changes moved to dev branch, ping me.

:)

I'm really excited about new release with includes minified html supports

**Comment by stephanebachelier** [comment]
@arthurvr @sindresorhus coming back. Sorry for the delay
